plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -38,6 +39,7 @@
 
 public final class JdbcTableHandle
         extends BaseJdbcConnectorTableHandle
+        implements ConnectorTableLocation
```

Comment:
Shouldn't we have a SPI specific API in `ConnectorMetadata` which would resolve location based on `ConnectorTableHandle` ?